docs/changelog.md

```diff
@@ -40,7 +40,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  | sc_types.EDGE_ACCESS_CONST_POS_TEMP | sc_type.CONST_TEMP_POS_ARC    |
+  | sc_types.EDGE_ACCESS_CONST_NEG_TEMP | sc_type.CONST_TEMP_NEG_ARC    |
+  | sc_types.EDGE_ACCESS_CONST_FUZ_TEMP | sc_type.CONST_FUZ_ARC         |
+  | sc_types.EDGE_U_COMMON_VAR          | sc_type.VAR_COMMON_EDGE       |
+  | sc_types.EDGE_D_COMMON_VAR          | sc_type.VAR_COMMON_ARC        |
+  | sc_types.EDGE_ACCESS_VAR_POS_PERM   | sc_type.VAR_PERM_POS_ARC      |
+  | sc_types.EDGE_ACCESS_VAR_NEG_PERM   | sc_type.VAR_PERM_NEG_ARC      |
+  | sc_types.EDGE_ACCESS_VAR_FUZ_PERM   | sc_type.VAR_FUZ_ARC           |
+  | sc_types.EDGE_ACCESS_VAR_POS_TEMP   | sc_type.VAR_TEMP_POS_ARC      |
+  | sc_types.EDGE_ACCESS_VAR_NEG_TEMP   | sc_type.VAR_TEMP_NEG_ARC      |
+  | sc_types.EDGE_ACCESS_VAR_FUZ_TEMP   | sc_type.VAR_FUZ_ARC           |
+  | sc_types.NODE_CONST                 | sc_type.CONST_NODE            |
+  | sc_types.NODE_VAR                   | sc_type.VAR_NODE              |
+  | sc_types.LINK                       | sc_type.NODE_LINK             |
+  | sc_types.LINK_CLASS                 | sc_type.NODE_LINK_CLASS       |
+  | sc_types.NODE_STRUCT                | sc_type.NODE_STRUCTURE        |
+  | sc_types.LINK_CONST                 | sc_type.CONST_NODE_LINK       |
+  | sc_types.LINK_CONST_CLASS           | sc_type.CONST_NODE_LINK_CLASS |
+  | sc_types.NODE_CONST_TUPLE           | sc_type.CONST_NODE_TUPLE      |
+  | sc_types.NODE_CONST_STRUCT          | sc_type.CONST_NODE_STRUCTURE  |
+  | sc_types.NODE_CONST_ROLE            | sc_type.CONST_NODE_ROLE       |
+  | sc_types.NODE_CONST_NO_ROLE         | sc_type.CONST_NODE_NO_ROLE    |
+  | sc_types.NODE_CONST_CLASS           | sc_type.CONST_NODE_CLASS      |
+  | sc_types.NODE_CONST_MATERIAL        | sc_type.CONST_NODE_MATERIAL   |
+  | sc_types.LINK_VAR                   | sc_type.VAR_NODE_LINK         |
+  | sc_types.LINK_VAR_CLASS             | sc_type.VAR_NODE_LINK_CLASS   |
+  | sc_types.NODE_VAR_STRUCT            | sc_type.VAR_NODE_STRUCTURE    |
+  | sc_types.NODE_VAR_TUPLE             | sc_type.VAR_NODE_TUPLE        |
+  | sc_types.NODE_VAR_ROLE              | sc_type.VAR_NODE_ROLE         |
+  | sc_types.NODE_VAR_NO_ROLE           | sc_type.VAR_NODE_NO_ROLE      |
+  | sc_types.NODE_VAR_CLASS             | sc_type.VAR_NODE_CLASS        |
+  | sc_types.NODE_VAR_MATERIAL          | sc_type.VAR_NODE_MATERIAL     | 
+  
+  From now on, all sc-links are sc-nodes. Types of actual and inactual temporal membership sc-arc were added.
+  Module `sc_types` is deprecated, use `sc_type` instead.
+
+- Type `ScType.NodeAbstract` was removed.
+
 ### Added
+ - ScType methods: `is_connector`, `is_structure`, `is_no_role`, `is_actual`, `is_inactual`, `is_superclass`
+ - All possible combinations of subtypes into sc-types
+ - Type `sc_type.NODE_SUPERCLASS`
+ - Types: `sc_type.CONNECTOR`, `sc_type.ARC`
+ - Types of actual and inactual temporal sc-arcs into sc-types
```

Comment:
```suggestion
 - All possible combinations of subtypes into sc-type
 - Type `sc_type.NODE_SUPERCLASS`
 - Types: `sc_type.CONNECTOR`, `sc_type.ARC`
 - Types of actual and inactual temporal sc-arcs into sc-type
```

Reply:
there were 2 suggestions in this comment

Reply:
I removed the second `in sc-types`